.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,21 @@
+---
+name: Pull Request
+about: Create a pull request to contribute to the TRD
+labels: 
+
+---
+IMPORTANT NOTICE: This PR complies with the [guidelines for contribution to the TRD](https://tezos-reward-distributor-organization.github.io/tezos-reward-distributor/contributers.html) and is qualified for being compensated by the TRD grant if approved by the maintainers.
```

Comment:
and MAY qualify for a compensation

Reply:
"this project started in the spirit of free and open source software with voluntary contributions without compensation. You can always donate your contribution free of charge"